grpclog/glogger/glogger.go

```diff
@@ -37,38 +37,64 @@ Package glogger defines glog-based logging for grpc.
 package glogger
```

Comment:
How about a usage comment for this package?  "Import this package anonymously to activate it as the enabled grpc logger" or something?